packages/avocado_framework.rb

```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Avocado_framework < Package
```

Comment:
Can we not rename this package?  Why make it harder for users by typing the extra `_framework`?  It also prevents updates for users that have already installed `avocado`.

Reply:
Avocado is ambiguous and may refer to other packages

Reply:
What makes it ambiguous?  And what does may refer to other packages mean?  Can you provide an example?

Reply:
Plus avocado is broken (the binaries refer to an older version of python) so I don't think anyone is using it.

Reply:
Can we keep avocado and just convert it to a fake package that depends on avocado_framework?

Reply:
Done.